src/standalone/Vegetation/canopy_parameterizations.jl

```diff
@@ -157,8 +158,10 @@ function compute_absorbances!(
     frac_diff,
 ) where {FT}
     RTP = RT.parameters
-    @. p.canopy.radiative_transfer.par = plant_absorbed_pfd(
-        RT,
+    @. p.canopy.radiative_transfer.par = plant_absorbed_pfd_two_stream(
```

Comment:
That is correct. Previously RT was passed into the function, and the type was used for dispatching to the correct function, before unpacking the params. I was not able to broadcast plant_absorbed_pfd with RT as an argument because RT contains a field. Instead I passed in the parameters used by plant_absorbed_pfd as arguments to avoid using RT or RTP as arguments.